launcher/src/components/UI/the-control/AmsterdamComponent.vue

```diff
@@ -74,49 +70,8 @@ export default {
     return {
       showSyncInfo: false,
       counter: 0,
-      networkIcon: "",
-      mainnetIcon: "/img/icon/control/mainnetIconControl.png",
-      testnetIcon: "/img/icon/control/testnetIconControl.png",
```

Comment:
I added this. you have to leave this in so the network icon changes accordingly 😉